tox.ini

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 120
```

Comment:
No action needed. This is cool - this is the first I've seen a tox.ini file used.